db/db.go

```diff
@@ -0,0 +1,46 @@
+package db
+
+import (
+	"fmt"
+
+	"google-scraper/models"
+
+	"github.com/beego/beego/v2/client/orm"
+	"github.com/beego/beego/v2/core/logs"
+	"github.com/beego/beego/v2/server/web"
+	_ "github.com/lib/pq"
+)
+
+func init() {
+	dbUrl, _ := web.AppConfig.String("dbUrl")
```

Comment:
@junan Since this file is used to bootstrap the application, this fix should be done. Not that you could create a chore task to tackle this. Just schedule it right after this PR or before this PR is merged.

Reply:
@olivierobert I pushed the changes in this PR. Do I need to do it in separate task?

Reply:
If you already made the changes in this PR, keep it by all means :-) I was just giving another option to close this PR without it.